service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ChildController.java

```diff
@@ -78,7 +79,7 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
                 .map(Child::getParty)
                 .map(Party::getDateOfBirth)
                 .filter(Objects::nonNull)
-                .filter(dateOfBirth -> dateOfBirth.isAfter(LocalDate.now()))
+                .filter(dateOfBirth -> dateOfBirth.isAfter(time.now().toLocalDate()))
```

Comment:
Do you think it would be worthwhile adding methods similar to what you did in the abstract controller class to the time interface? Could have a `dateNow()` that defaults to calling `now().toLocalDate()`

Reply:
Yup I will add dateNow() to the time interface.

Reply:
If we do this then we don't need the methods in the abstract controller class.

Reply:
Also, if you don't mind increasing the file count even more, maybe the name of `Time` could be better. From that name you might assume it doesn't deal with dates so maybe it is worth while renaming it to DateTime? Then it can have 3 methods
```java
public interface DateTime {
    LocalDateTime now();

    default LocalDate dateNow() {
        return now().toLocalDate();
    }

    default LocalTime timeNow() {
        return now().toLocalTime();
    }
}
```

Reply:
Actually thinking about it maybe leave it for a PR straight after this one has merged if you want to do it

Reply:
I would prefer to defer the change to another PR.